### PR TITLE
GET /api/articles (queries) for order and sort_by

### PR DESCRIPTION
**Pull request by nkrpradhan**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -213,6 +213,7 @@ describe("GET /api/articles with comment count", () => {
         expect(articles).toBeSorted({ key: "created_at", descending: true });
       });
   });
+
   test("Bad request", () => {
```
Comment (aloush-dev) [inline __tests__/app.test.js:217]: what is this test doing?

**Review by aloush-dev — APPROVED** [review]
You havent tested any of your error handling which I would ask you add.
Then, merge away!

File: __tests__/app.test.js (hunk 1 above)
Comment (nkrpradhan) [reply __tests__/app.test.js:217]: This is checking for default sorted and order. By default, the order is desc and sorted by date.

Comment (nkrpradhan) [reply __tests__/app.test.js:217]: Added the test cases now.

**Pull request merged**